ansible/roles/fips/tasks/redhat-8.yaml

```diff
@@ -0,0 +1,19 @@
+- name: "redhat 8 fips: ensure fips_enabled is 1"
+  lineinfile:
+    dest: /proc/sys/crypto/fips_enabled
+    line: "1"
+    state: present
+  become: yes
+  check_mode: yes
+  register: sysfips
+
+- name: "redhat 8 fips: run fips enable command"
+  command: fips-mode-setup --enable
+  when:
+    - sysfips is changed
+
+- name: "redhat 8 fips: reboot to pick up fips changes"
+  ansible.builtin.reboot:
+  when:
+    - sysfips is changed
+    - fips_reboot
```

Comment:
I don't think it makes sense to change the FIPS mode but not reboot. IMO it should be all or nothing.